addons/common/XEH_postInit.sqf

```diff
@@ -330,6 +330,18 @@ if (isServer) then {
         _object hideObjectGlobal _hide;
     }] call CBA_fnc_addEventHandler;
 
+    [QGVAR(setbuildingMarker), {
+        params ["_objects", "_setMarker"];
+
+        if !(_objects isEqualType []) then {
+            _objects = [_objects];
+        };
+
+        {
+            [_x, _setMarker] call FUNC(setbuildingMarker);
+        } forEach _objects;
```

Comment:
Put this in the function itself. The event should just be a simple `call` only wrapper.

Reply:
https://github.com/zen-mod/ZEN/pull/704/commits/df7388c87d42a6f409e1256b36478aa186d5e9f1#diff-904878a4c3022a6347e5b85fe824f3641b2911718a3985159b1d6bdf0b3ad5e0R21